content/03.methods.md

```diff
@@ -117,6 +131,18 @@ We annotated putative driver fusions and prioritized fusions lists with kinases,
 We also added chimerDB [@doi:10.1093/nar/gkw1083] annotations to both driver and prioritized fusion list.
 
 ### Clinical Data Harmonization
+#### WHO Classification of Disease Types
+
+The `disease_type_old` field in the `pbta-histologies.tsv` file contains the diagnosis denoted from the patient's pathology report.
+The `disease_type_new` field in the `pbta-histologies.tsv` file includes updates to `disease_type_old`, including any diagnosis denoted as "Other" as well as a changes on the basis of a molecular alteration.
+The `broad_histology` denotes the broad 2016 WHO classification [doi:10.1007/s00401-016-1545-1] for each tumor.
+The `short_histology` is an abbreviated version of the `broad_histology`.
+
+#### Molecular Subtyping
+Medulloblastoma subtypes SHH, MYC, Group 3, and Group 4 were predicted using an [RNA expression classifier](https://github.com/PichaiRaman/MedulloClassifier) on the RSEM FPKM data.
```

Comment:
The README on that repo does a nice job of discussing the results, but it'd be good to have a blurb about what the various scripts do. It also looks like a lot of them depend on data files that aren't included in the repo. I don't think it needs to be addressed before this PR is merged, but we should keep an eye on things. This won't be sufficient for publication, so hopefully there will be a preprint/etc that we can cite.

Reply:
You make reference to which columns are used in the histology file in `#### WHO Classification of Disease Types` section above. I know this is a bit of a different case but do you think there is value in noting what column this information is in at least for the moment?

Reply:
@PichaiRaman is writing up a paper he hopes to submit before his departure